database/README.md

```diff
@@ -201,7 +199,7 @@ This table outlines the primary key columns in the database and how they relate
 
 Because there can be a lag time of weeks, even months, before law enforcement investigators submit their crash report to TxDOT, the Vision Zero team needs the ability to manually create "temporary" records so that reporting metrics are more timely/accurate.
 
-The VZE makes this possible by allowing users to insert crash, unit, and people records directly into the database. User-created records must be inserted into the `_cris` tables to ensure the proper data flow across the `_edits` and unified tables.
+The VZE makes this possible by allowing users to insert crash, unit, and people records directly into the database. User-created records must be inserted into the `_cris` tables to ensure proper data flow into the unified tables.
```

Comment:
I don't know if it's more helpful to be more explicit and say that the DB expects that every single record that exists in the `_cris` tables also exists in the unified tables, and vice versa.  Or if "ensure proper data flow" covers it 🤷